scylla/src/transport/connection.rs

```diff
@@ -1292,6 +1248,59 @@ pub async fn open_named_connection(
     Ok((connection, error_receiver))
 }
 
+async fn start_authenticate(
```

Comment:
The name `start_authenticate` implies that the function starts the authentication process but does not finish it. How about `perform_authentication` instead?